sky/utils/common_utils.py

```diff
@@ -54,6 +54,15 @@ def get_user_hash():
     return user_hash
 
 
+def get_job_run_id(run_timestamp: str, cluster_name: Optional[str],
```

Comment:
I'm a bit confused. We have `job id`, `job run id`, `(SkyPilot) run id`, and `usage run id`... Can we make it easier to distinguish the names?

Reply:
To my understanding, `usage run id` can be called `usage id`, right?
And the first three can be just `job_id` (or `run id`), can't they?

Reply:
I'm also wondering whether the timestamp and the cluster name should be included in the job run id.

Reply:
> To my understanding, usage run id can be called usage id, right?

The `usage_run_id` means for each run of `sky launch`/`sky status`/..., it will generate a new hash for it, so that we can distinguish different runs in the dashboard. I think it may be better to keep it `usage_run_id`

> And the first three can be just job_id (or run id), can't they?

They are different: the `job_id` is the id in the job queue of a specific VM, `SKYPILOT_RUN_ID` is the id we used to distinguish the jobs globally across the clusters (refer to the following answer for why we have run timestamp, and cluster name in it).

> I'm also wondering whether the timestamp and the cluster name should be included in the job run id.

The `run_timestamp` and the `cluster_name` in the `SKYPILOT_RUN_ID` is used to distinguish the jobs globally across different clusters. For example, if the user has multiple clusters, `test-1`, `test-2` for hyperparameter tuning, and uploads the logs to the wandb using the `SKYPILOT_RUN_ID`. If the run_timestamp and cluster_name are not included, they will both log to `id-1`.


That said above, here is the proposal:
1. Keep `usage_run_id`
2. Replace `run_timestamp` with `job_timestamp`
3. Change the name `get_job_run_id` with `get_global_job_id`

Wdyt?

Reply:
Thanks for the detailed explanation! I see. I like your proposal.